libr/core/anal_objc.c

```diff
@@ -214,6 +214,7 @@ static bool objc_find_refs(RCore *core) {
 		if (core->anal->verbose) {
 			eprintf ("Could not find necessary Objective-C sections...\n");
 		}
+		core_objc_free (objc);
```

Comment:
its crasahing because https://github.com/radareorg/radare2/pull/17313 is not merged. ill rebase when this one gets in